src/ts/mapboxMap.ts

```diff
@@ -220,20 +213,17 @@ module powerbi.extensibility.visual {
                     'line_string': true     // Lasso is overriding the 'line_string' mode
                 },
             });
-            console.log(this.settings.api.zoom)
-
 
-            // console.log('geocoder', this.geocoder)
+            this.geocoder = new MapboxGeocoder({
+                accessToken: this.settings.api.accessToken,
+                zoom: 10
```

Comment:
We should add proximity to the default search, which we can do by adding the `options.trackProximity` value to true from https://github.com/mapbox/mapbox-gl-geocoder/blob/master/API.md#mapboxgeocoder